### DISCUSSION TITLE
Using font variants in apps, e.g. Excel

### DISCUSSION THREAD

**Discussion by JNLlB** [character-map-uwp/Character-Map-UWP#266] [Q&A] [answered]
We use a special corporate font which has a few stylistic sets. How can I use, say, tabular figures (in my case Stylistic Set 5) in an app like Excel? Is there any way within Character Map UWP to specify the default set to be used by Windows or Office? Or of modifying the font and installing a second copy incorporating the variations I want? This is what I have at the moment:
  
 
![image](https://github.com/character-map-uwp/Character-Map-UWP/assets/41243440/e016371d-57ee-4547-bff7-b296fd380a4c)

**Comment by JohnnyWestlake** [comment, marked as answer]
Excel doesn't support OpenType typography features unfortunately (although Word does). 

You would probably have to use some dedicated font editing software to change this unfortunately, though I don't have any good free suggestions.

**Comment by JNLlB** [comment]
Thank you, Johnny. That's sad. 
I'll have a look at using [FontForge](https://fontforge.org/en-US/) to create a copy of the font with the default glyphs replaced by their tabular variants - unless you could code Character Map to provide this functionality ...
